zebra/zebra_ptm.c

```diff
@@ -598,6 +600,19 @@ static int zebra_ptm_handle_msg_cb(void *arg, void *in_ctxt)
 		return 0;
 	}
 
+	ptm_lib_find_key_in_msg(in_ctxt, ZEBRA_PTM_BFDVRF_STR, vrf_str);
+
+	if (vrf_str[0] == '\0') {
+		zlog_debug("%s: Key %s not found in PTM msg", __func__,
```

Comment:
unguarded debugs?

Reply:
actually, this is not an error. this can happen to not have vrf in the message.
but the error will be that default vrf does not exist, which needs to return -1.
so I made some changes here